src/lib/viewers/error/PreviewErrorViewer.js

```diff
@@ -59,17 +59,15 @@ class PreviewErrorViewer extends BaseViewer {
         this.setup();
 
         const { file, showDownload } = this.options;
-        this.icon = ICON_FILE_DEFAULT;
+        this.icon = getIconFromName('FILE_DEFAULT');
 
         // Generic errors will not have the file object
         if (file) {
             switch (file.extension) {
                 case 'zip':
                 case 'tgz':
-                    this.icon = ICON_FILE_ZIP;
-                    break;
                 case 'flv':
-                    this.icon = ICON_FILE_MEDIA;
+                    this.icon = getIconFromExtension(file.extension);
```

Comment:
This is here to maintain current behavior. Eventually we can just rely on the setup on base viewer and then fall back if needed